<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="originalFrame">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>570</width>
      <height>410</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Circle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagePathEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="imgLoadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="processingButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Processing Image</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Gaussian</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Hand</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Erode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilate</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textInfor">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="trackerButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tracker</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenCV_Test">
    <property name="title">
     <string>OpenCV Test</string>
    </property>
   </widget>
   <addaction name="menuOpenCV_Test"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
